.github/workflows/e2e.generic.tag.main.goreleaser.slsa3.yml

```diff
@@ -0,0 +1,142 @@
+on:
+  schedule:
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # # triggers only if push new tag version, like `v0.8.4` or else
+
+permissions: read-all
+
+env:
+  PAT_TOKEN: ${{ secrets.E2E_GENERIC_TOKEN }}
+  GH_TOKEN: ${{ github.token }}
+  ISSUE_REPOSITORY: slsa-framework/slsa-github-generator
+  DEFAULT_VERSION: v18.0.0
```

Comment:
make sure to pick a unique default version. Check if this one is already used in another workflow.

Reply:
thanks